.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
   pull_request:
 
 env:
-  NODE_VERSION: 18
+  NODE_VERSION: 18.17
```

Comment:
Can be removed when Node 18.18.1 is released. See https://github.com/embroider-build/embroider/issues/1532.